assets/optional/cert-manager/release-cert-manager-x86_64.json

```diff
@@ -0,0 +1,13 @@
+{
+  "release": {
+    "base": "4.19"
```

Comment:
Why is this 4.19?

Reply:
http://registry.redhat.io/redhat/redhat-operator-index:v4.19 is the latest that is published to the catalog, so there is no `openshift-cert-manager-operator` in http://registry.redhat.io/redhat/redhat-operator-index:v4.20